features/aragon/votingLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { Link } from '@lidofinance/lido-ui';
+import { FC, PropsWithChildren, memo, useEffect, useState } from 'react';
+import { validateNumericInput } from '../../shared/ui/inputValidators';
+
+export const VotingLink: FC<PropsWithChildren<{ voteId?: string }>> = memo(
+  ({ voteId, children }) => {
+    const [href, setHref] = useState('https://vote.lido.fi');
+
+    useEffect(() => {
+      const url = new URL(window.location.href);
+      const tld =
+        url.hostname === 'localhost'
+          ? 'lido.fi'
+          : url.hostname.split('.').slice(-2).join('.');
```

Comment:
IDK why my same comment on this topic disappeared, but again... will it work with branch-preview?

Reply:
<img width="957" alt="image" src="https://github.com/lidofinance/trp-ui/assets/103929444/acba6960-559c-443e-9081-d69eb7ca8b28">

It shows for me as outdated, maybe that's why

Reply:
I would prefer to have some switch here with default fallback to lido.fi

Reply:
Makes sense, will add